boards/firefly/board.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2017 DAI Labor TU Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    boards_firefly
```

Comment:
copy-paste, defgroup should be used once in board.h, here only use `@ingroup  boards_firefly`